docs/Development Concepts/Protocols/2.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Protocol Identifiers
+---
+
+Currently the main protocol identifiers used for Waku are:
+
+- using `WAKU2-RELAY`/`WAKU2-FILTER` nodes for live message publish/subscribe on a content topic
+  - `WAKU2-FILTER` is a lightweight version of `WAKU2-FILTER` specifically designed for bandwidth restricted devices.
+  - For more information, you can read the specifications for [`WAKU2-FILTER`](https://rfc.vac.dev/spec/12/) and [`WAKU2-RELAY`](https://rfc.vac.dev/spec/11/)
+- using `waku-store` nodes to save/retrieve persisting messages
+
+### Waku Relay: `/vac/waku/relay/2.0.0`
+
+`WAKU2-RELAY` specifies a Publish/Subscribe approach to peer-to-peer messaging with a strong focus on privacy, censorship-resistance, security and scalability.
+Its current implementation is a minor extension of the libp2p GossipSub protocol and prescribes gossip-based dissemination.
+
+### Waku Filter: `/vac/waku/filter/2.0.0-beta1`
+
+`WAKU2-FILTER` is a protocol that enables subscribing to messages that a peer receives.
+This is a more lightweight version of `WakuRelay` specifically designed for bandwidth restricted devices. This is due to the fact that light nodes subscribe to full-nodes and only receive the messages they desire.
```

Comment:
I would not say it's a lightweight version of relay, instead I would say that filter enables a node to access the relay network without the connectivity and bandwith requirements of relay but it comes with privacy drawbacks.

We can then explain the drawbacks in a the detailed section

Reply:
refactored
the [RFC](https://rfc.vac.dev/spec/12/) mentions it as a "lightweight version of relay"

Reply:
also, where can we find more info on the comparisons/drawbacks for this?

Reply:
> refactored the [RFC](https://rfc.vac.dev/spec/12/) mentions it as a "lightweight version of relay" so was trying to keep consistency

Oh, right I see. Hum. Ok I guess it's fine then :)